declarations_site/spotter/utils.py

```diff
@@ -27,6 +28,10 @@
 logger = logging.getLogger(__name__)
 
 
+class DjangoStorage(BaseStorage):
```

Comment:
that's all, previous implementation was inherited two base classes, and then simplified for one. It was left for future possible use.